winston-logger/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
```

Comment:
could you rename the file to `winston-logger.js`?
I was thinking that if someone check out the project and will search for "index" it will be a mess 😂